aws/table_aws_drs_source_server.go

```diff
@@ -0,0 +1,166 @@
+package aws
+
+import (
+	"context"
+	"math"
+
+	"github.com/aws/aws-sdk-go-v2/service/drs"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+func tableAwsDRSSourceServer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_drs_source_server",
+		Description: "AWS DRS Source Server",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "source_server_id", Require: plugin.Optional},
+			},
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"UninitializedAccountException"}),
+			},
+			Hydrate: listAwsDRSSourceServer,
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "source_server_id",
+				Description: "The ID of the Source Server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SourceServerID"),
+			},
+			{
+				Name:        "arn",
+				Description: "The ARN of the Source Server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "recovery_instance_id",
+				Description: "The ID of the Recovery Instance associated with this Source Server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RecoveryInstanceId"),
+			},
+			{
+				Name:        "source_properties",
+				Description: "The source properties of the Source Server.",
+				Type:        proto.ColumnType_JSON,
```

Comment:
This table has a lot of JSON columns, so when writing example docs, please be sure to create examples showcasing how to dig into key JSON columns, e.g., https://hub.steampipe.io/plugins/turbot/mongodbatlas/tables/mongodbatlas_cluster#get-auto-scaling-details-of-all-clusters